.travis.yml

```diff
@@ -9,14 +9,14 @@ go_import_path: github.com/kubernetes/kompose
 
 matrix:
   include:
-    - go: 1.7
+    - go: "1.9"
```

Comment:
mind removing the quotes? otherwise this LGTM!

Reply:
Travis requires (or used to require, this may be fixed by now) minor versions with two digits (10, 11, etc.) to have quotes, or use the format 1.10.x and 1.11.x.

Otherwise it incorrectly parses them as

    1.10 -> 1.1
    1.11 -> 1.1

when starting the builds.

Reply:
ah that makes sense. no worries then!